activerecord/test/schema/schema.rb

```diff
@@ -664,6 +664,11 @@
     t.string   :name
   end
 
+  create_table :service_contracts, force: true do |t|
```

Comment:
Are there any particular reason why this `create_table` statement is not sorted alphabetically as outlined in the start of this file https://github.com/rails/rails/blob/bead3221c706e468e3625b71b7e99e9fd333a80c/activerecord/test/schema/schema.rb#L4-L9 ?

Reply:
No reason - I missed that. Thanks.